CONTRIBUTING.md

```diff
@@ -364,7 +363,7 @@ $ docker run -v $(pwd):/app -w /app aws-cdk <CDK ARGS>
 In many cases, you don't really need to build the entire project. Say you want to work on the `@aws-cdk/aws-ec2` module:
 
 ```console
-$ ./install.sh
+$ ./build.sh
```

Comment:
It's not really approriate to replace `install.sh` with `build.sh` here. The goal of this section is to only build a specific package and its dependencies.

Looks like a simple `yarn install` would do the trick

Reply:
@nmussy  Thank you for your advice. I'll replace it.